tensorflow/python/keras/utils/vis_utils.py

```diff
@@ -86,6 +103,11 @@ def model_to_dot(model,
     expand_nested: whether to expand nested models into clusters.
     dpi: Dots per inch.
     subgraph: whether to return a `pydot.Cluster` instance.
+    layer_range: range of layer names passed as `list` of `str` of shape (2,)
```

Comment:
So to make it more readable, we expect an input of string list with two items, which is the starting layer name, and ending layer name (I guess both inclusive)?

Reply:
done